src/gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -998,12 +1029,19 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  tumor_burden:
+    $ref:
+      - "_terms.yaml#/tumor_burden/common"
+    type: number
+    minimum: 0
```

Comment:
I did not see a minimum defined in DICT-453

Reply:
Updated ticket, thanks